### ISSUE TITLE
Unable to return to server screen

### ISSUE THREAD

**Issue by jbruchanov** [symless/synergy#5397]
PC1 - Server (win10x64)
PC2 - Client (win10x64).

I rebooted PC1, while PC2 was still running.
When i reached login screen on PC1 i typed incorrectly password and moved mouse accidentally to PC2.
Cursor and keyboard has been lock to PC2 only and i couldn't go back to PC1 screen to retype my password again...

Stopping PC2 helped to obtain control of PC1 back.

**Comment by nlyan** [comment]
This sounds like a problem with elevate mode. Have you tried ticking the 'Elevate' checkbox inside Synergy settings before starting the server/client?

Changes to elevation will be made in 1.8.x

**Comment by kirashi3** [comment]
As of Synergy 1.8.1-beta-f7377d6, this issue still occurs with the login screen, and more over UAC prompts.
Server: Windows 10 x64
Client: Windows 10 x64
Elevate mode is enabled on both client and server under settings.

**Comment by XinyuHou** [comment]
Duplicate of #5075
